app/assets/javascripts/pageflow/editor/views/uploader_view.js

```diff
@@ -14,51 +10,42 @@ pageflow.UploaderView = Backbone.Marionette.View.extend({
 
     this.bindUIElements();
 
-    this.$el.fileupload({
-      acceptFileTypes: new RegExp ('(\\.|\\/)(bmp|gif|jpe?g|png|ti?f|wmv|mp4|mpg|mov|asf|asx|avi|' +
-                                   'm?v|mpeg|qt|3g2|3gp|3ivx|divx|3vx|vob|flv|dvx|xvid|mkv|vtt)$',
-                                   'i'),
-      dataType: 'json',
-
+    var form = this.$el;
+    var fileInput = this.$('input:file');
```

Comment:
Naming the selector in the `ui` section and referencing it as `this.ui.fileInput` would be the more Backbone idiomatic approach here. Why do we need to to pass the input explicitly now?

Reply:
That's just my old jQuery way of thinking here. Still getting used to Backbone paradigms.
I figured out the `el` from the backbone docs but not the `ui` part. Caught up now by looking at other js files within the project